docs/commerce-api.yaml

```
@@ -0,0 +1,250 @@
+openapi: 3.0.0
+info:
+  title: Commerce Project API
+  description: 상품 조회, 검색, 주문 및 결제 기능을 제공하는 API
+  version: 1.0.0
+  contact:
+    name: DEV
+    email: aslanhobae@gmail.com
+
+servers:
+  - url: https://commerce.aslan.com/v1
+    description: 운영 서버
+  - url: https://localhost:8080/v1
+    description: 로컬 환경
+
+tags:
+  - name: 상품
+    description: 상품 관련 API
+  - name: 주문&결제
+    description: 주문 및 결제 관련 API
+
+paths:
+  /products:
```

```suggestion
  /v1/products:
```
버저닝 붙이고 시작하는게 좋은 습관입니다